sdk/communication/Azure.Communication.CallingServer/tests/CallConnection/CallConnectionLiveTests.cs

```diff
@@ -79,12 +85,366 @@ public async Task RunCreateAddRemoveHangupScenarioTests()
 
             try
             {
-                string userId = GetFixedUserId("0000000d-5a5f-2db9-ccd7-44482200049a");
+                string userId = GetFixedUserId(TestEnvironment.UserIdentifier);
+                await WaitForOperationCompletion().ConfigureAwait(false);
+
+                // Add Participant
+                AddParticipantResult addParticipantResult = await AddParticipantOperation(callConnection, userId).ConfigureAwait(false);
+                Assert.NotNull(addParticipantResult);
+
+                await WaitForOperationCompletion().ConfigureAwait(false);
+
+                // Remove Participant
+                await RemoveParticipantOperation(callConnection, userId).ConfigureAwait(false);
+            }
+            catch (RequestFailedException ex)
+            {
+                Console.WriteLine(ex.Message);
+                Assert.Fail($"Unexpected error: {ex}");
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex.Message);
+                Assert.Fail($"Unexpected error: {ex}");
+            }
+            finally
+            {
+                // Hang up the Call, there is one call leg in this test case, hangup the call will also delete the call as the result.
+                await WaitForOperationCompletion().ConfigureAwait(false);
+                await HangupOperation(callConnection).ConfigureAwait(false);
+            }
+        }
+
+        [Test]
+        public async Task RunCreateAddMuteGetParticipntRemoveHangupScenarioTests()
+        {
+            if (SkipCallingServerInteractionLiveTests)
+                Assert.Ignore("Skip callingserver interaction live tests flag is on.");
+
+            CallingServerClient client = CreateInstrumentedCallingServerClientWithConnectionString();
+
+            // Establish a call
+            var callConnection = await CreateCallConnectionOperation(client).ConfigureAwait(false);
+
+            try
+            {
+                string userId = GetFixedUserId(TestEnvironment.UserIdentifier);
+                await WaitForOperationCompletion().ConfigureAwait(false);
```

Comment:
move this to the line above it, as we are waiting for the callconnection operation. Also pl move the call creation to inside the try block for this and other tests